config/routes.rb

```diff
@@ -45,6 +45,7 @@
       resources :confirm_publish_course, only: %i[edit update], path: "confirm-publish-course"
 
       resource :course_details, concerns: :confirmable, only: %i[edit update], path: "/course-details"
+      resource :lead_school, concerns: :confirmable, only: %i[edit update], path: "/lead-school"
```

Comment:
This confirm route will also have the trainees `employing_school` on (if applicable), so it should probably be in a separate `schools` route.

Let's tackle this on the next ticket!

I'll add a comment to https://trello.com/c/ko1NKaqW/1567-s-add-employing-school-to-confirm-page